analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -255,7 +257,8 @@ abstract class PackageManager(
                     val errors = listOf(
                         createAndLogIssue(
                             source = managerName,
-                            message = "Resolving dependencies for '${definitionFile.name}' failed with: " +
+                            message = "Resolving $managerName dependencies for " +
+                                    "'${definitionFile.relativeTo(analysisRoot)}' failed with: " +
```

Comment:
I believe you also need `.invariantSeparatorsPath` here for the test on Windows to succeed.